src/test/java/tools/jackson/databind/WriteDatesAsTimestampsTests.java

```diff
@@ -0,0 +1,242 @@
+package tools.jackson.databind;
```

Comment:
Should probably go in `tools.jackson.databind.ext.javatime.ser` (under `src/test`) -- possibly splitting `java.util.Date` / `Calendar` into `tools.jackson.databind.ser.jdk`.